<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Day8Window</class>
 <widget class="QDialog" name="Day8Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>899</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,0">
   <property name="topMargin">
    <number>30</number>
   </property>
   <property name="rightMargin">
    <number>30</number>
   </property>
   <property name="bottomMargin">
    <number>30</number>
   </property>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>100</height>
      </size>
     </property>
     <property name="text">
      <string>They can be old boyz too, living up to 50 years!!! No way, that do be nuts lol.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>100</height>
      </size>
     </property>
     <property name="text">
      <string>Geckos have microscopic hair on their feet!!</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_4">
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>100</height>
      </size>
     </property>
     <property name="text">
      <string>Also, you know those funky frills that some lizards have, those are used to scare off predators!</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>100</height>
      </size>
     </property>
     <property name="text">
      <string>The biggest lizard is the Komodo dragon and these boys can be up to 2.6 meters long!!</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="day8_title_label">
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>300</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>Day 8: Some cool facts about lizards for ya!</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
